game/core/src/main/java/com/gg/game/Model/Vector.java

```diff
@@ -45,6 +49,20 @@ public Vector scale(float scale) {
         return new Vector(x * scale, y * scale);
     }
 
+    public static Vector normalize(Vector oldVector){
+        if (oldVector.getLength() != 0) {
+            Vector newVector = new Vector(oldVector.getX() / oldVector.getLength(), oldVector.getY() / oldVector.getLength());
```

Comment:
```suggestion
            Vector newVector = new Vector(
                oldVector.getX() / oldVector.getLength(),
                oldVector.getY() / oldVector.getLength());
```
Readability suggestion